python/ray/train/tests/test_mosaic_trainer.py

```diff
@@ -22,10 +21,9 @@ def trainer_init_per_worker(config):
     import composer.optim
 
     BATCH_SIZE = 32
-    # prepare the model for distributed training and wrap with ComposerClassifier for
-    # Composer Trainer compatibility
-    model = config.pop("model", torchvision.models.resnet18(num_classes=10))
-    model = ComposerClassifier(ray.train.torch.prepare_model(model))
+    model = ComposerClassifier(
+        config.pop("model", torchvision.models.resnet18(num_classes=10))
```

Comment:
If we don't call `prepare_model`, does Mosaic still perform distributed training? 

IIRC if you don't call `prepare_model` in normal Torch, the program trains `num_workers` independent models (as opposed to one distributed model).

Reply:
Composer library automatically checks the world size and rank and applies distributed training